.github/workflows/android.yml

```diff
@@ -67,19 +67,6 @@ jobs:
         run: |
           echo ANDROID_NDK_ROOT="" >> $GITHUB_ENV
 
-      - name: Build manager
-        if: ${{ success() && matrix.type == 'manager' }}
-        run:  |
-          ./android/ci_build_manager.sh
```

Comment:
This and other files are not used anymore but still present it the repo. Please do a cascade removal of all outdated scripts

Reply:
Clean the scripts can done in other PR.

Reply:
Why not to clean them in one PR since they are not used anymore?

Reply:
There more scripts for android that can remove, and they need to remove and discuss in different PR.

Reply:
I kindly ask you to remove outdated scripts in this PR